components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/APIMgtGoogleAnalyticsTrackingHandler.java

```diff
@@ -83,40 +80,25 @@ public boolean handleRequest(MessageContext msgCtx) {
             span = Util.startSpan(APIMgtGatewayConstants.GOOGLE_ANALYTICS_HANDLER, responseLatencySpan, tracer);
         }
         try {
-            if (configKey == null) {
```

Comment:
@ChamodDamitha  this contains backward incompatible change shall we fix this with backward compatibility

Reply:
Fixed